docs/en/ingest-management/elastic-agent/otel-agent.asciidoc

```diff
@@ -0,0 +1,145 @@
+[[otel-agent]]
+= Run {agent} as an OTel Collector
+
+preview::[]
+
+The link:https://opentelemetry.io/docs/collector/[OpenTelemetry Collector] is a vendor-neutral way to receive, process, and export telemetry data. {agent} includes an embedded OTel Collector, enabling you to instrument your applications and infrastructure once, and send data to multiple vendors and backends. 
+
+When you run {agent} in `otel` mode it supports the standard OTel Collector configuration format that defines a set of receivers, processors, exporters, and connectors. Logs, metrics, and traces can be ingested using OpenTelemetry data formats.
+
+The following steps will help you to get set up quickly using {agent} as an OTel Collector. Note that all steps are run with `sudo` as it's generally recommended to run {agent} with administrative privileges.
+
+[discrete]
+== Run {agent} in OTel mode
+
+For simplicity, these steps demonstrate OTel mode using standalone {agent}. The install steps for {fleet}-managed {agent} are very similar. Refer to the <<elastic-agent-installation,Install {agents}>> section for the full set of requirements and steps for any install scenario.
+
+. On your host, download and extract the {agent} installation package. 
++
+--
+// tag::install-elastic-agent[]
+
+include::{ingest-docs-root}/docs/en/ingest-management/tab-widgets/download-widget.asciidoc[]
+
+// end::install-elastic-agent[]
+--
++
+The commands shown are for AMD platforms, but ARM packages are also available.
+Refer to the {agent} https://www.elastic.co/downloads/elastic-agent[downloads page]
+for the full list of available packages.
+
+. Change into the agent package directory and run {agent} with the `otel` subcommand and the `-h` option to view the command syntax and available parameters:
++
+[source,shell]
+--
+cd <agent-package-name>
+sudo ./elastic-agent otel -h
+--
++
+TIP: You can also refer to the <<elastic-agent-otel-command,{agent} OTel>> command documentation for the full command syntax.
+
+. View or update the OTel configuration file.
++
+A basic OTel configuration file named `otel.yml` is included in the {agent} package. You can view or update it using any text editor.
++
+[source,yaml]
+--
+receivers:
+  filelog:
+    include: [ /var/log/system.log ]
+    start_at: beginning
+
+processors:
+  resource:
+    attributes:
```

Comment:
instead of using the resource processor we should use the transform processor:

```
  log_statements:
    - context: resource
      statements:
        - keep_keys(attributes, ["service.name", "service.namespace", "cloud.region"])
    - context: log
      statements:
        - set(severity_text, "FAIL") where body == "request failed"
        - replace_all_matches(attributes, "/user/*/list/*", "/user/{userId}/list/{listId}")
        - replace_all_patterns(attributes, "value", "/account/\\d{4}", "/account/{accountId}")
        - set(body, attributes["http.route"])
```

Reply:
Note to self: see https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/processor/transformprocessor/README.md